TranCalculator.cpp:39:27: error: ‘numeric_limits’ is not a member of ‘std’

I am trying to build `opm-common` on Ubuntu 21.04, gcc 10.3.0 : 

```
$ git clone git@github.com:OPM/opm-common.git
$ cd opm-common/
$ mkdir build
$ cmake ..
-- The C compiler identification is GNU 10.3.0
-- The CXX compiler identification is GNU 10.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Deprecation Warning at cmake/Modules/OpmInit.cmake:26 (cmake_policy):
  The OLD behavior for policy CMP0026 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
Call Stack (most recent call first):
  CMakeLists.txt:56 (OpmSetPolicies)


CMake Warning at cmake/Modules/Findopm-tests.cmake:26 (message):
  opm-tests not found - integration tests using opm-tests will be skipped.
Call Stack (most recent call first):
  CMakeLists.txt:60 (include)


-- Performing Test HAVE_C99
-- Performing Test HAVE_C99 - Success
-- Found C99: -std=c99  
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found suitable version "1.74.0", minimum required is "1.44.0") found components: system unit_test_framework 
CMake Warning (dev) at /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:273 (message):
  The package name passed to `find_package_handle_standard_args` (CJSON) does
  not match the name of the calling package (cjson).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/Modules/Findcjson.cmake:77 (find_package_handle_standard_args)
  cmake/Modules/OpmFind.cmake:135 (find_package)
  cmake/Modules/OpmPackage.cmake:189 (find_and_append_package_to)
  opm-common-prereqs.cmake:25 (find_package_deps)
  CMakeLists.txt:65 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find CJSON (missing: CJSON_INCLUDE_DIRS HAVE_CJSON) 
-- Performing Test HAVE_ATTRIBUTE_ALWAYS_INLINE
-- Performing Test HAVE_ATTRIBUTE_ALWAYS_INLINE - Success
-- Performing Test HAS_ATTRIBUTE_UNUSED
-- Performing Test HAS_ATTRIBUTE_UNUSED - Success
-- Performing Test HAS_ATTRIBUTE_DEPRECATED
-- Performing Test HAS_ATTRIBUTE_DEPRECATED - Success
-- Performing Test HAS_ATTRIBUTE_DEPRECATED_MSG
-- Performing Test HAS_ATTRIBUTE_DEPRECATED_MSG - Success
-- CMake version: 3.18.4
-- Linux distribution: Ubuntu 21.04
-- Target architecture: x86_64
-- Found Git: /bin/git (found version "2.30.2") 
-- Source code repository: git c05ea0fe2
-- GNU C++ compiler version: 10.3.0
-- Linker: ld 2.36.1
Processing opm_defaults opm-common
-- Checking to see if CXX compiler accepts flag -Wl,--enable-new-dtags
-- Checking to see if CXX compiler accepts flag -Wl,--enable-new-dtags - yes
-- Precompiled headers: disabled
-- Build type: Release
-- Checking to see if CXX compiler accepts flag -mtune=native
-- Checking to see if CXX compiler accepts flag -mtune=native - yes
-- All warnings enabled: -Wall -Wextra -Wshadow
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Checking to see if CXX compiler accepts flag -pthread
-- Checking to see if CXX compiler accepts flag -pthread - yes
-- Could NOT find CppCheck (missing: CPPCHECK_PROGRAM) 
-- Disabling clang-check as CMAKE_EXPORT_COMPILE_COMMANDS is not enabled
CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  cmake/Modules/OpmFind.cmake:135 (find_package)
  cmake/Modules/OpmFind.cmake:227 (find_and_append_package_to)
  cmake/Modules/OpmLibMain.cmake:83 (find_and_append_package_list_to)
  CMakeLists.txt:222 (include)


CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  cmake/Modules/OpmFind.cmake:135 (find_package)
  cmake/Modules/OpmFind.cmake:227 (find_and_append_package_to)
  cmake/Modules/OpmLibMain.cmake:83 (find_and_append_package_list_to)
  CMakeLists.txt:222 (include)


-- Found Boost: /usr/include (found suitable version "1.74.0", minimum required is "1.44.0") found components: system unit_test_framework 
-- Generating debug symbols: -ggdb3
-- Looking for strip utility
-- Looking for strip utility - found
CMake Warning at /usr/share/cmake-3.18/Modules/FindBoost.cmake:1187 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1311 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.18/Modules/FindBoost.cmake:1919 (_Boost_MISSING_DEPENDENCIES)
  cmake/Modules/UseDynamicBoost.cmake:1 (find_package)
  cmake/Modules/OpmLibMain.cmake:99 (include)
  CMakeLists.txt:222 (include)


-- Performing Test HAVE_DYNAMIC_BOOST_TEST
-- Performing Test HAVE_DYNAMIC_BOOST_TEST - Success
-- Writing config file "/home/hakon/test/opm-common/build/config.h"...
-- This build defaults to installing in /usr/local
-- Checking whether the file system is case-sensitive
-- File system is case-sensitive
-- Found Doxygen: /bin/doxygen (found version "1.9.1") found components: doxygen dot 
-- Writing version information to local header project-version.h
-- Configuring done
-- Generating done
-- Build files have been written to: /home/hakon/test/opm-common/build
$ make
Scanning dependencies of target update-version
[  0%] Updating version information
[  0%] Built target update-version
Scanning dependencies of target genkw
[  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/json/JsonObject.cpp.o
[  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/createDefaultKeywordList.cpp.o
[  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/UDAValue.cpp.o
[  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/DeckValue.cpp.o
[  0%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/Deck.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/DeckItem.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/DeckKeyword.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/DeckRecord.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Deck/DeckOutput.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Generator/KeywordGenerator.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Generator/KeywordLoader.cpp.o
[  1%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/ErrorGuard.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/ParseContext.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/ParserEnums.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/ParserItem.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/ParserKeyword.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/ParserRecord.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/raw/RawKeyword.cpp.o
[  2%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/raw/RawRecord.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Parser/raw/StarToken.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Units/Dimension.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/parser/eclipse/Units/UnitSystem.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/utility/OpmInputError.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/OpmLog.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/Logger.cpp.o
[  3%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/StreamLog.cpp.o
[  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/LogBackend.cpp.o
[  4%] Building CXX object CMakeFiles/genkw.dir/src/opm/common/OpmLog/LogUtil.cpp.o
[  4%] Building C object CMakeFiles/genkw.dir/external/cjson/cJSON.c.o
[  4%] Linking CXX executable bin/genkw
[  4%] Built target genkw
[  5%] Generating tmp_gen/ParserKeywords/A.cpp, tmp_gen/ParserKeywords/B.cpp, tmp_gen/ParserKeywords/C.cpp, tmp_gen/ParserKeywords/D.cpp, tmp_gen/ParserKeywords/E.cpp, tmp_gen/ParserKeywords/F.cpp, tmp_gen/ParserKeywords/G.cpp, tmp_gen/ParserKeywords/H.cpp, tmp_gen/ParserKeywords/I.cpp, tmp_gen/ParserKeywords/J.cpp, tmp_gen/ParserKeywords/K.cpp, tmp_gen/ParserKeywords/L.cpp, tmp_gen/ParserKeywords/M.cpp, tmp_gen/ParserKeywords/N.cpp, tmp_gen/ParserKeywords/O.cpp, tmp_gen/ParserKeywords/P.cpp, tmp_gen/ParserKeywords/Q.cpp, tmp_gen/ParserKeywords/R.cpp, tmp_gen/ParserKeywords/S.cpp, tmp_gen/ParserKeywords/T.cpp, tmp_gen/ParserKeywords/U.cpp, tmp_gen/ParserKeywords/V.cpp, tmp_gen/ParserKeywords/W.cpp, tmp_gen/ParserKeywords/X.cpp, tmp_gen/ParserKeywords/Y.cpp, tmp_gen/ParserKeywords/Z.cpp, tmp_gen/ParserInit.cpp, tmp_gen/TestKeywords.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/A.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/B.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/C.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/D.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/E.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/F.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/G.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/H.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/I.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/J.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/K.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/L.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/M.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/N.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/O.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/P.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/Q.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/R.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/S.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/T.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/U.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/V.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/W.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/X.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/Y.cpp
Updated source file written to /home/hakon/test/opm-common/build/tmp_gen/ParserKeywords/Z.cpp
Updated init file written to /home/hakon/test/opm-common/build/tmp_gen/ParserInit.cpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/A.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/B.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/C.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/D.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/E.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/F.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/G.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/H.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/I.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/J.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/K.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/L.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/M.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/N.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/O.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/P.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/Q.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/R.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/S.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/T.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/U.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/V.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/W.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/X.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/Y.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/Z.hpp
Updated header file written to /home/hakon/test/opm-common/build/tmp_gen/include/opm/parser/eclipse/Parser/ParserKeywords/Builtin.hpp
[  5%] Generating ParserKeywords/A.cpp, ParserKeywords/B.cpp, ParserKeywords/C.cpp, ParserKeywords/D.cpp, ParserKeywords/E.cpp, ParserKeywords/F.cpp, ParserKeywords/G.cpp, ParserKeywords/H.cpp, ParserKeywords/I.cpp, ParserKeywords/J.cpp, ParserKeywords/K.cpp, ParserKeywords/L.cpp, ParserKeywords/M.cpp, ParserKeywords/N.cpp, ParserKeywords/O.cpp, ParserKeywords/P.cpp, ParserKeywords/Q.cpp, ParserKeywords/R.cpp, ParserKeywords/S.cpp, ParserKeywords/T.cpp, ParserKeywords/U.cpp, ParserKeywords/V.cpp, ParserKeywords/W.cpp, ParserKeywords/X.cpp, ParserKeywords/Y.cpp, ParserKeywords/Z.cpp, TestKeywords.cpp, ParserInit.cpp
Scanning dependencies of target opmcommon
[  5%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/Z.cpp.o
[  5%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/Y.cpp.o
[  5%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/X.cpp.o
[  5%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/W.cpp.o
[  5%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/V.cpp.o
[  5%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/U.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/T.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/S.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/R.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/Q.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/P.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/O.cpp.o
[  6%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/N.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/M.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/L.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/K.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/J.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/I.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/H.cpp.o
[  7%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/G.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/F.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/E.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/D.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/C.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/B.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserKeywords/A.cpp.o
[  8%] Building CXX object CMakeFiles/opmcommon.dir/ParserInit.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/CounterLog.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/EclipsePRTLog.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/LogBackend.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/Logger.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/LogUtil.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/KeywordLocation.cpp.o
[  9%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/InfoLogger.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/OpmLog.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/StreamLog.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/OpmLog/TimerLog.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/ActiveGridCells.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/FileSystem.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/numeric/MonotCubicInterpolator.cpp.o
[ 10%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/OpmInputError.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/Parameter.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/ParameterGroup.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/parameters/ParameterTools.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/numeric/calculateCellVol.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/common/utility/TimeService.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/io/eclipse/SummaryNode.cpp.o
[ 11%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/json/JsonObject.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/Deck.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/DeckItem.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/DeckValue.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/DeckKeyword.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/DeckRecord.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/DeckOutput.cpp.o
[ 12%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/DeckSection.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/ImportContainer.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Deck/UDAValue.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Python/Python.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/Action/PyAction.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/AquiferConfig.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/AquiferCT.cpp.o
[ 13%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/Aquifetp.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/Aquancon.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/AquiferHelpers.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquiferCell.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquiferConnection.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/SingleNumericalAquifer.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquifers.cpp.o
[ 14%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/checkDeck.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/EclipseConfig.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/EclipseState.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/EndpointScaling.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/FieldPropsManager.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/Box.cpp.o
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/BoxManager.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/FaceDir.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/FaultCollection.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/Fault.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/FaultFace.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/GridDims.cpp.o
[ 16%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.cpp.o
[ 17%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/NNC.cpp.o
[ 17%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/Operate.cpp.o
[ 17%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/PinchMode.cpp.o
[ 17%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp.o
[ 17%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/setKeywordBox.cpp.o
[ 17%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp.o
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp: In member function ‘Opm::Fieldprops::keywords::keyword_info<double> Opm::Fieldprops::TranCalculator::make_kw_info(Opm::Fieldprops::ScalarOperation)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp:39:27: error: ‘numeric_limits’ is not a member of ‘std’
   39 |         kw_info.init(std::numeric_limits<double>::max());
      |                           ^~~~~~~~~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp:39:42: error: expected primary-expression before ‘double’
   39 |         kw_info.init(std::numeric_limits<double>::max());
      |                                          ^~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp:42:27: error: ‘numeric_limits’ is not a member of ‘std’
   42 |         kw_info.init(std::numeric_limits<double>::lowest());
      |                           ^~~~~~~~~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp:42:42: error: expected primary-expression before ‘double’
   42 |         kw_info.init(std::numeric_limits<double>::lowest());
      |                                          ^~~~~~
make[2]: *** [CMakeFiles/opmcommon.dir/build.make:2505: CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:3339: CMakeFiles/opmcommon.dir/all] Error 2
make: *** [Makefile:160: all] Error 2

--- a/opm/parser/eclipse/EclipseState/Grid/TranCalculator.hpp
+++ b/opm/parser/eclipse/EclipseState/Grid/TranCalculator.hpp
@@ -28,7 +28,7 @@ namespace Opm
 namespace Fieldprops
 {
 
-namespace keywords { template<class T> class keyword_info; }
+namespace keywords { template<class T> struct keyword_info; }
 
 enum class ScalarOperation {
     ADD = 1,

--- a/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp
+++ b/src/opm/parser/eclipse/EclipseState/Grid/TranCalculator.cpp
@@ -20,6 +20,8 @@
 #include <opm/parser/eclipse/EclipseState/Grid/TranCalculator.hpp>
 #include <opm/parser/eclipse/EclipseState/Grid/Keywords.hpp>
 
+#include <limits>
+
 namespace Opm
 {
 

